dev-tools/mage/kibana.go

```diff
@@ -25,15 +25,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+const kibanaBuildDir = "build/kibana"
+
 // KibanaDashboards collects the Kibana dashboards files and generates the
 // index patterns based on the fields.yml file. It outputs to build/kibana.
 // Use PackageKibanaDashboardsFromBuildDir() with this.
 func KibanaDashboards(moduleDirs ...string) error {
-	var kibanaBuildDir = "build/kibana"
-
-	if err := os.RemoveAll(kibanaBuildDir); err != nil {
```

Comment:
Is this cleanup not needed anymore?

Reply:
Good catch. I've add this back. Merging issue.